### DISCUSSION TITLE
Cesium.ExperimentalFeatures.enableModelExperimental 设置报错，请问如何打开这个设置

### DISCUSSION THREAD

**Discussion by bingolovess** [zouyaoji/vue-cesium#555] [General]
![image](https://github.com/zouyaoji/vue-cesium/assets/46583511/16c94ac2-c3ef-4736-b186-586ddb2702e8)
此写法报错，请问如何打开，现在是 tileset._customShader = customShader;自定义着色器需要打开

**Comment by zouyaoji** [comment]
如果用的vue-cesium的组件，应该是加到VcPrimitiveTileset组件上吧。

也可以拿到tileset用原生API设置。
